build/commands/lib/l10nUtil.js

```diff
@@ -156,6 +156,31 @@ module.exports.ethereumRemoteClientPaths = [
   '../../../ethereum-remote-client/brave/app/_locales/en/messages.json'
 ]
 
+// Helper function to retrieve Greaselion script paths relative to the
+// Brave paths.
+module.exports.getGreaselionScriptPaths = function () {
+  const jsonContent = fs.readFileSync('../../../brave-site-specific-scripts/Greaselion.json', 'utf8')
+  if (!jsonContent) {
+    console.error('Missing Greaselion.json')
+    return []
+  }
+
+  const jsonObject = JSON.parse(jsonContent)
+  if (!jsonObject) {
+    console.error('Malformed Greaselion.json')
+    return []
+  }
+
+  let paths = []
+  jsonObject.forEach((item) => {
+    if (item.messages) {
+      paths.push(`../../../brave-site-specific-scripts/${item.messages}/en_US/messages.json`)
```

Comment:
perhaps allow the local path of the brave-site-specific-scripts repo to be passed in as a cli flag?

Reply:
Good idea, added a new commit that supports this option.

Reply:
I think you forgot to use `basePath` for this line (190)?

Reply:
Yes, thanks for catching that - fixed.